src/test/java/util/xcmailr/AbstractTest.java

```diff
@@ -72,25 +37,33 @@ private static void deleteTempFile(File tempFile)
             {
                 Files.delete(tempFile.toPath());
             }
-            catch (Exception e)
+            catch (final Exception e)
             {
-                System.out.println(MessageFormat.format("Couldn''t delete temporary file: ''{0}'' caused by {1}",
-                                                        tempFile.getAbsolutePath(), e));
+                // double apostrophe needed, otherwise MessageFormat.format() won't work
+                System.out.println(MessageFormat.format(
+                                                        "Coundn''t delete temporary file: ''{0}'' caused by {1}",
+                                                        tempFile.getAbsoluteFile(), e));
```

Comment:
```suggestion
                System.out.println(MessageFormat.format("Couldn''t delete temporary file: ''{0}'' caused by {1}",
                                                        tempFile.getAbsoluteFile(), e));
```
Spelling & Formatting